db/migrate/20200525072638_remove_none_tags.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class RemoveNoneTags < ActiveRecord::Migration[6.0]
+  def up
+    none_tag = Tag.where("lower(name) = ?", "none").first
```

Comment:
We should be using sql here instead of using the AR models. It's annoying because there are so many associations to delete... This case is so simple it might be ok.

Reply:
I think it's better if we iterate over the associations `[:tag_users, :topic_tags, ... ]` and delete the rows with sql. `delete from #{table_name} where tag_id = :none_tag_id`

Reply:
Update the migration to use SQL.